### PR TITLE
add PeriPlex to the projects

### LINKED ISSUE #180: 021-Periplex: Add Project Page

### PR DESCRIPTION
**Pull request by AgatheZ**
- This PR adds PeriPlex to the website projects

### CODE REVIEW

**Review by hshuaib90 — CHANGES_REQUESTED** [review]
Looks great except that the summary is from the dental project :) Could you change that then I can approve

**Comment by hshuaib90** [comment]
also @heyhaleema in case you want to add any comments

File: _projects/AIC021-PeriPlex.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+status: Developing
+image: /assets/img/blog/nhs_logo.png
+summary: AI-powered algorithm for categorising patients based on their perioperative risk, enabling appropriate pathway planning and optimisation in elective surgeries.
+csc-lead: <a href="/team_member/Haleema.html">Haleema</a>
+modality: Perioperative patient complexity assessment
+pathology: Patient risk stratification for elective surgery
+rationale: The current process for determining patient suitability for elective surgery is fragmented and occurs close to the procedure date, leading to cancellations and delays for high-risk patients.
+patient-pathway: Currently, suitability for surgery is determined at the pre-assessment stage, which occurs 1-2 days before the scheduled procedure. This process is not optimised and lacks information from various sources.
+training-data: N/A
+errors:  Delayed optimisation for high-risk patients can lead to increased morbidity, mortality, and reduced patient outcomes. Inappropriate stratification can also lead to increased waiting times for low-risk patients.
```
Comment (heyhaleema) [inline _projects/AIC021-PeriPlex.md]: @AgatheZ @hshuaib90 Just a minor suggestion, but no pressure to add if unnecessary.

``` suggestion
errors:  Delayed optimisation of surgery planning for high-risk patients can lead to increased morbidity, mortality, and negative patient outcomes. Inappropriate stratification can also lead to increased waiting times for both low and high-risk patients, and inefficient allocation of clinical resources.
```

**Review by heyhaleema — APPROVED** [review]
@AgatheZ I've committed a minor change to update the spelling from US to UK for consistency, but otherwise looks great to me! 

I've also made a minor suggestion for an edit to the errors description, but this isn't required i.e., happy for you to commit the suggestion or ignore and resolve the conversation.

@hshuaib90 Happy to merge this once you've approved the changes.

**Comment by AgatheZ** [comment]
Thanks both for the feedback/suggestions, I've made the requested changes.

**Review by hshuaib90 — APPROVED** [review]

**Pull request merged**